Finjector.Web/ClientApp/src/components/Details/DetailsAeErrors.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert } from "reactstrap";
+
+interface DetailsAeErrorsProps {
+  errors: string[] | undefined;
+  hasWarnings: boolean | undefined;
+  warnings: string[] | undefined;
+}
+
+const DetailsAeErrors = (props: DetailsAeErrorsProps) => {
+  const { errors, hasWarnings, warnings } = props;
```

Comment:
seems kinda jumbled here, like you have warnings and hasWarnings, and all are optional.  shouldn't you be able to tell if there are warnings by if there are elements in the warnings array?

Reply:
hasWarnings is something returned from the AE query, i'm not sure if there's a case where it would be true and the warnings array would be false. not sure what to do in that case, obviously we can't show the alerts, but what should the little colored icon that says "VALID" or "WARNING" or "ERROR" say?

Reply:
It should show valid and warning

Reply:
https://finjector-test.azurewebsites.net/details/K302300049-TASK01-ADNO006-770000